velox/connectors/hive/HiveConfig.cpp

```diff
@@ -243,6 +243,17 @@ bool HiveConfig::s3UseProxyFromEnv() const {
   return config_->get<bool>(kS3UseProxyFromEnv, false);
 }
 
+uint8_t HiveConfig::parquetReadTimestampUnit(const Config* session) const {
+  const auto unit = session->get<uint8_t>(
+      kParquetReadTimestampUnitSession,
+      config_->get<uint8_t>(kParquetReadTimestampUnit, 9 /*nano*/));
```

Comment:
As far as I know the default is millisecond

Reply:
Changed to millisecond as the default value. Thanks.